### ISSUE TITLE
Remove the 'default' layout

### ISSUE THREAD

**Issue by triblondon** [Financial-Times/o-grid#46]
Browsers that do not support media queries should stack all grid containers vertically, so that we encourage a more 'mobile-first' design.  Desktop UAs that lack decent media query support, but still have a significant user base can be locked to a particular layout at a fixed width, however those that do not support box-sizing cannot do this, so notably IE6 and 7 will display the site in a single column view.

IE7 does have max-width, so we can at least limit the width of the column when no media queries are active.  In IE6, there's a hack available to apply max-width that we could include, probably in a oUseragentTarget.

So, in summary:

* IE9+ is responsive
* IE8 gets a fixed 900px ish wide layout, non-responsive
* IE7 and 6 get a single column 'mobile' layout
* This is nothing to do with core/enhanced experience / cuts the mustard, because it is all CSS.   There's no JS involved here.

@danskinner are you OK with this?

**Comment by kaelig** [comment]
Removed this feature with #69.

- IE 6, 7 get a broken layout (should be easily fixed by loading a `box-sizing` polyfill), but the site should still be readable. Products could not stick to that rule anyway (imagine having to put a media query around each width to make sure these old browsers don't get the layout rules!).
- IE 8 gets the fixed layout at 970px (great for 1024px wide screens)
- Modern browsers get the responsive view
